src/Event.php

```diff
@@ -640,7 +640,7 @@ public function toArray(): array
             $data['breadcrumbs']['values'] = $this->breadcrumbs;
         }
 
-        if (null !== $this->exceptions) {
+        if (!empty($this->exceptions)) {
```

Comment:
I would say that this statement can just be removed since we would just loop over an empty array and do nothing, so it's useless.

Reply:
I have followed the standard of `fingerprint` and` breadcrumbs`
Do you want me to discard this PR and create another?

Reply:
>I have followed the standard of `fingerprint` and `breadcrumbs`

Nope, it's different because in this case we would loop over the array and append each item to the `$data['exception']['values']` variable (implicitly creating the `$data['exception']` key), but since it would be empty it's a noop. In the other cases we're not looping but directly assigning the values to the `$data` array, so both keys would be set even if there is no need to

>Do you want me to discard this PR and create another?

Nope, no reason to do it :smiley:

Reply:
yes of course, but I did not want to make too many changes to the code, even because I'm using the github editor.
But I'll be more careful next time.

Reply:
I saw you marked the issue as resolved, but it's not. Please update the PR with the requested changes so that we can merge it 😉

Reply:
oops... I don't use github pr, sorry, how can I append/merge a commit in existing pr ?

Reply:
Just checkout the branch of this PR, edit the file, commit and push like always. Or you can edit the file again from the GitHub interface like I suppose you did, just be sure to edit the one in the branch [`patch-1`](https://github.com/tonybolzan/sentry-php/tree/patch-1)

Reply:
ok... Can I combine `array_reverse` inside of `foreach`, create a new variable `$reversedException` for this don't make sense for me.

Reply:
Sure, feel free to make as much changes as you like if they makes sense and do not deviate too much from the purpose of the issue. Also remember to re-request the review when done, otherwise the reviewers may miss that more changes have been pushed if they receive a lot of notifications